src/toil/provisioners/clusterScaler.py

```diff
@@ -542,7 +530,7 @@ def setNodeCount(self, nodeType, numNodes, preemptable=False, force=False):
                                                                    preemptable=preemptable,
                                                                    force=force)
                 else:
-                    logger.info('Cluster already at desired size of %i. Nothing to do.', numNodes)
+                    logger.info('Cluster (minus ignored nodes) already at desired size of %i. Nothing to do.', numNodes)
```

Comment:
Should this have a different log message if `force` is set?

Reply:
I wouldn't be against it, but I'm not sure what it'd look like. What kind of message were you thinking?

Reply:
Just the original message, since `delta = numNodes - (numCurrentNodes - numIgnoredNodes)` (and thus `(minus ignored nodes)`) is only True if != force.

Reply:
ah OK, that makes sense.